DynamicTablesPkg/Library/Common/SsdtPcieSupportLib/SsdtPcieOscTemplate.asl

```diff
@@ -46,21 +46,16 @@ DefinitionBlock ("SsdtPciOsc.aml", "SSDT", 2, "ARMLTD", "PCI-OSC", 1) {
 
       // Save Capabilities DWord2 & 3
       Store (CDW2, SUPP)
-      Store (CDW3, CTRL)
+      And (CTRL, CDW3, CTRL)
 
       // Only allow native hot plug control if OS supports:
       // * ASPM
       // * Clock PM
       // * MSI/MSI-X
       If (LNotEqual (And (SUPP, 0x16), 0x16)) {
-        And (CTRL, 0x1E, CTRL) // Mask bit 0 (and undefined bits)
+        And (CTRL, 0x9E, CTRL) // Mask bit 0 (and undefined bits)

```

Comment:
Dumb question: what is this extra bit for? Do we need to update the comment?

Reply:
The extra bit (bit 7) is for granting native DPC control to the OS